custom_components/plugwise/binary_sensor.py

```diff
@@ -287,6 +287,9 @@ def _async_process_data(self):
 
             for notify_id, details in notify.items():
                 for msg_type, msg in details.items():
+                    if msg_type not in SEVERITIES:
+                        msg_type = "other"
+
                     self._attributes[msg_type.upper()] = msg
```

Comment:
This should say: `self._attributes[f"{msg_type.upper()}_msg"].append(msg)`

Reply:
Lacking copy/paste skills today

Reply:
:)